### DISCUSSION TITLE
[Feature]: Show/use timestamps with original timezone

### DISCUSSION THREAD

**Discussion by AndreAle94** [immich-app/immich#1589] [Feature Request]
### Feature detail

It would be nice to show the timestamp of the media (and group them) using the timezone where the picture has been taken, instead of using the user/server specific one. It could be useful when you take pictures in a different timezone.

---
Example: I'm currently based in Italy and the last summer I went to USA for vacation where I took some photos/videos. When I check the details tab, all the timestamps are converted to the local timezone.

Some details:
- Exif metadata (Date/Time Original): 2022:08:30 12:27:47.456-07:00
- Italian TZ is currently GTM+1 (location of the server/user)
- Arizona TZ was GTM-7 (location where I took the picture)

Current UI:
![immagine](https://user-images.githubusercontent.com/15969021/216773861-5d38964c-fb55-472b-8e84-57e0aa37626d.png)
Note: the UI is still using a wrong timezone

I would like to show the original timestamp:
```
30 ago 2022
mar, 12:27 GMT-07:00
```
---
The timeline is currently grouping media using the user (or server) timezone, not the original one. Another example, I took a picture in Los Angeles at (Date/Time Original) 2022:09:01 20:09:15.542286-07:00 but this image is shown in the group of Sept-02

### Platform

Web

**Comment by Spectre5** [comment]
+1 I would also appreciate this

**Comment by AndreAle94** [comment]
For reference, this feature is partially implemented by #1894 (first point)

**Comment by uhthomas** [comment]
Should be fixed by #2908.

**Comment by RobinFrcd** [comment]
My photo has `Date/Time Original : 2024:04:05 09:59:29.713+10:00`  but Web UI shows `5 Apr 2024, Fri, 00:59 GMT+01:00`. Is there a setting somewhere to show the original TZ instead of the one of the server ?
